pkg/services/mysql/all_in_one_arm_template.go

```diff
@@ -78,7 +51,7 @@ var allInOneARMTemplateBytes = []byte(`
 					"type": "firewallrules",
 					"apiVersion": "[variables('DBforMySQLapiVersion')]",
 					"dependsOn": [
-						"[concat('Microsoft.DBforMySQL/servers/', parameters('serverName'))]"
+						"[concat('Microsoft.DBforMySQL/servers/', variables('ServerName'))]"
 					],
 					"location": "[parameters('location')]",
```

Comment:
We currently do tags/location into the map at the deployer layer. We'll need to do a refactor there to switch fully over to go params. should be easy enough to do, but we'll need to bump all the arm templates.